steps/test/unit/tReorderMSData.cc

```diff
@@ -21,6 +22,30 @@ using dp3::steps::Step;
 
 BOOST_AUTO_TEST_SUITE(reorder)
 
+bool compareMetaBinaries(const std::string& filename1,
+                         const std::string& filename2) {
+  std::ifstream file1(
+      filename1,
+      std::ifstream::ate | std::ifstream::binary);  // open file at the end
+  std::ifstream file2(
+      filename2,
+      std::ifstream::ate | std::ifstream::binary);  // open file at the end
+
+  if (file1.fail() || file2.fail()) {
+    std::cout << "Failed to open Meta file" << std::endl;
+  }
+
+  file1.ignore(std::numeric_limits<std::streamsize>::max(),
+               '\n');  // Ignore first line
+  file2.ignore(std::numeric_limits<std::streamsize>::max(),
+               '\n');  // Ignore first line
```

Comment:
There should be some kind of check after the ignore to make sure that the entire file has not accidentally being ignored and that a comparison is indeed still taking place, e.g. check that the remaining length is > 0